### ISSUE TITLE
Document Formatting: no beeps as indentation indicators while browsing in 2023.1

### ISSUE THREAD

**Issue by rmcpantoja** [nvaccess/nvda#14802]
### Steps to reproduce:

1. You can configure indentation in NVDA menu, settings, and choose Document Formatting.
2. Then, navigate the window and change the value of Line indentation reporting to Both Speech and Tones.
3. Apply changes.
4. You can browse something that has indentations, ex: a text, a code, etc. You will notice that the beeps don't play.

## code example using tabs:

import num2words
def convert_num_to_words(utterance, language):
	digits = re.findall(r'\d', utterance)
	if len(digits ) > 29:
		tones.beep(1200, 100)
		utterance =_(
			# Translators: Error message when the number is too big.
			_("The number is too big! twenty seven numbers maximum")
	)
	else:
		utterance = ' '.join([num2words.num2words(i ,lang=language) if i.isdigit() else i for i in utterance.split()])
		if not utterance[0].isupper():
			utterance = utterance.capitalize()
	return utterance

### Actual behavior:

NVDA doesn't make any beeps while we navigate through, for example, this code.

### Expected behavior:

NVDA should beeps while we navigate through the document.

### NVDA logs, crash dumps and other attachments:

INFO - __main__ (10:56:52.301) - MainThread (4284):
Starting NVDA version 2023.1
INFO - core.main (10:56:52.355) - MainThread (4284):
Config dir: C:\Users\LENOVO_User\AppData\Roaming\nvda
INFO - config.ConfigManager._loadConfig (10:56:52.360) - MainThread (4284):
Loading config: C:\Users\LENOVO_User\AppData\Roaming\nvda\nvda.ini
INFO - core.main (10:56:52.488) - MainThread (4284):
Windows version: Windows 11 22H2 (10.0.22621) workstation AMD64
INFO - core.main (10:56:52.488) - MainThread (4284):
Using Python version 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:01:55) [MSC v.1900 32 bit (Intel)]
INFO - core.main (10:56:52.488) - MainThread (4284):
Using comtypes version 1.1.11
INFO - core.main (10:56:52.488) - MainThread (4284):
Using configobj version 5.1.0 with validate version 1.0.1
INFO - NVDAHelperLocal (10:56:52.706) - MainThread (4284):
Thread 4284, build\x86\localWin10\oneCoreSpeech.cpp, ocSpeech_initialize, 215:
ocSpeech_initialize

INFO - NVDAHelperLocal (10:56:52.706) - MainThread (4284):
Thread 4284, build\x86\localWin10\oneCoreSpeech.cpp, OcSpeechState::activate, 89:
Activating

INFO - NVDAHelperLocal (10:56:52.745) - MainThread (4284):
Thread 4284, build\x86\localWin10\oneCoreSpeech.cpp, preventEndUtteranceSilence_, 443:
AppendedSilence supported

INFO - synthDriverHandler.setSynth (10:56:52.755) - MainThread (4284):
Loaded synthDriver oneCore
WARNING - mathPres.initialize (10:56:52.760) - MainThread (4284):
MathPlayer 4 not available
INFO - core.main (10:56:52.760) - MainThread (4284):
Using wx version 4.1.1 msw (phoenix) wxWidgets 3.1.5 with six version 1.16.0
INFO - brailleInput.initialize (10:56:52.765) - MainThread (4284):
Braille input initialized
INFO - braille.initialize (10:56:52.765) - MainThread (4284):
Using liblouis version 3.24.0
INFO - braille.initialize (10:56:52.765) - MainThread (4284):
Using pySerial version 3.5
INFO - braille.BrailleHandler.setDisplayByName (10:56:52.769) - MainThread (4284):
Loaded braille display driver 'noBraille', current display has 0 cells.
INFO - core.main (10:56:52.884) - MainThread (4284):
Java Access Bridge support initialized
INFO - UIAHandler.UIAHandler.MTAThreadFunc (10:56:52.945) - UIAHandler.UIAHandler.MTAThread (7460):
UIAutomation: IUIAutomation6
INFO - NVDAHelperLocal (10:56:53.040) - UIAHandler.UIAHandler.MTAThread (7460):
Thread 7460, build\x86\UIARemote\UIARemote.cpp, initialize, 145:
Registered C:\Program Files (x86)\NVDA\lib\2023.1\Microsoft.UI.UIAutomation.dll.manifest

INFO - NVDAHelperLocal (10:56:53.040) - UIAHandler.UIAHandler.MTAThread (7460):
Thread 7460, build\x86\UIARemote\UIARemote.cpp, initialize, 150:
Microsoft.UI.UIAutomation is available

INFO - external:globalPlugins.speechLogger.GlobalPlugin.applyUserConfig (10:56:53.105) - MainThread (4284):
No log directory set. Disabling.
INFO - external:globalPlugins.numpadNavMode.GlobalPlugin.__init__ (10:56:53.129) - MainThread (4284):
numpadNavMode: initialized numpad to Windows mode.
INFO - core.main (10:56:53.394) - MainThread (4284):
NVDA initialized
INFO - external:globalPlugins.speechLogger.GlobalPlugin.applyUserConfig (10:56:53.555) - MainThread (4284):
No log directory set. Disabling.
INFO - external:globalPlugins.remoteClient.GlobalPlugin.on_connected_as_slave (10:56:54.315) - MainThread (4284):
Control connector connected
ERROR - speech.manager.SpeechManager._handleIndex (10:56:58.598) - MainThread (4284):
Error running speech callback
Traceback (most recent call last):
  File "speech\manager.pyc", line 685, in _handleIndex
  File "speech\commands.pyc", line 360, in run
TypeError: beep() got an unexpected keyword argument 'isSpeechBeepCommand'
ERROR - speech.manager.SpeechManager._handleIndex (10:57:01.696) - MainThread (4284):
Error running speech callback
Traceback (most recent call last):
  File "speech\manager.pyc", line 685, in _handleIndex
  File "speech\commands.pyc", line 360, in run
TypeError: beep() got an unexpected keyword argument 'isSpeechBeepCommand'
ERROR - speech.manager.SpeechManager._handleIndex (10:57:04.061) - MainThread (4284):
Error running speech callback
Traceback (most recent call last):
  File "speech\manager.pyc", line 685, in _handleIndex
  File "speech\commands.pyc", line 360, in run
TypeError: beep() got an unexpected keyword argument 'isSpeechBeepCommand'

#### NVDA installed/portable/running from source:

installed

#### NVDA version:

2023.1

#### Windows version:

Windows 11 22H2 (AMD64) build 22621.1413

#### Name and version of other software in use when reproducing the issue:

This is unnecessary, since indentation occurs in a general way.

#### Other information about your system:

* 8 cores
* 12 GB RAM

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

Yes, before updating to 2023 the beeps for indentation occurred normally.

#### If NVDA add-ons are disabled, is your problem still occurring?

As unbelievable as it sounds, the problem is fixed, so it could be a add-on. However, I can't find any indication as to which one may be causing the problem.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

No, just restart with addons disabled.

**Comment by CyrilleB79** [comment]
You can check disabling add-ons one by one. Or more quickly by dichotomy, i.e. beginning with half of the add-ons disabled and then enable/disabling by halves.

**Comment by Adriani90** [comment]
Could you please upload the whole log file so we can see which addons could cause this?

Do you by chance have the addon global plugin extensions enabled?

**Comment by CyrilleB79** [comment]
Pasting this command in the console may reveal the add-on responsible of this:
`import tones;tones.beep.__module__`

**Comment by rmcpantoja** [comment]
Hi @CyrilleB79 and @Adriani90, 
Thanks to your suggestions I've discovered the add-on that causes this problem, and it is about [enhanced tones for NVDA](https://github.com/davidacm/enhancedTones).
I've set the log to "debug", but I found no clues, not even with tones.beep.module. I think it's because NVDA's speech_manager object is replaced with a modified speech_manager, sort of like I've done in my new numbers to words add-on.

**Comment by CyrilleB79** [comment]
@rmcpantoja so you just need to close this ticket since it is not an issue with NVDA's code.
Please contact the add-on's author instead.

**Comment by Adriani90** [comment]
cc: @davidacm

**Comment by davidacm** [comment]
Hi, I'm aware of this issue, but haven't had the time to update my add-on. I apologize for this and for not reporting it sooner.

I will try to solve it as soon as possible.
Regards,
David CM.

**Comment by davidacm** [comment]
Hi, I just fixed the issue with enhanced tones. Get the latest release here:
https://davidacm.github.io/getlatest/gh/davidacm/EnhancedTones

**Comment by XLTechie** [comment]
@davidacm Can you submit to 
https://github.com/nvaccess/addons-datastore/issues/new 
please, so more users have access to it?
